### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by elkizana**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ A curated list of awesome things and projects built by Algerian developpers.
 - [Algerial Logos](https://github.com/mohsenuss91/AlgerianLogos) - Some Algerian official vectorized (.SVG) Logos
 
 ## Applications
+- [Salat Time](https://github.com/Barbarosse/salat_time/) A simple python Web App that returns the five Islamic Prayer Times.
```
Comment (youknowriad) [inline README.md]: Hi! thanks for the PR. The list is alphabetically ordered. Can we move this to the right place?

Comment (elkizana) [reply README.md]: Sorry, I didn't notice it.

Comment (youknowriad) [reply README.md]: It's fine :) I think you forgot to remove it from the initial position.

**Review by elkizana — COMMENTED** [review]
put in alphabetical order

**Comment by youknowriad** [comment]
I see that it's there twice now. I think you probably forgot to remove it from the initial position.

**Review by youknowriad — APPROVED** [review]
LGTM 👍

**Pull request merged**